jest/setup.js

```diff
@@ -105,6 +105,12 @@ const mockNativeModules = {
   FbRelayNativeAdapter: {
     updateCLC: jest.fn(),
   },
+  Geolocation: {
+    getCurrentPosition: jest.fn(),
+    watchPosition: jest.fn(),
+    clearWatch: jest.fn(),
```

Comment:
One question though, I looked at [this definition](https://github.com/facebook/react-native/blob/master/Libraries/Geolocation/Geolocation.js#L59) to decide on the methods to mock, but you say I should only mock the native APIs. Should I completely omit the `Geolocation` JS class, and only mock the native methods? (then, I would have to mock `getCurrentPosition()` and `stopObserving()` on the `LocationObserver` too.